test/type_check/type_error/formatter_test.exs

```diff
@@ -5,36 +5,46 @@ defmodule TypeCheck.TypeError.FormatterTest do
 
   import TypeCheck.Type.StreamData
 
-  for module <- [TypeCheck.Builtin.Atom,
-                 TypeCheck.Builtin.Binary,
-                 TypeCheck.Builtin.Bitstring,
-                 TypeCheck.Builtin.Boolean,
-                 TypeCheck.Builtin.FixedList,
-                 TypeCheck.Builtin.FixedMap,
-                 TypeCheck.Builtin.FixedTuple,
-                 TypeCheck.Builtin.Float,
-                 TypeCheck.Builtin.Integer,
-                 TypeCheck.Builtin.Lazy,
-                 TypeCheck.Builtin.List,
-                 TypeCheck.Builtin.Literal,
-                 TypeCheck.Builtin.Map,
-                 TypeCheck.Builtin.NamedType,
-                 TypeCheck.Builtin.NegInteger,
-                 TypeCheck.Builtin.NonNegInteger,
-                 TypeCheck.Builtin.None,
-                 TypeCheck.Builtin.Number,
-                 TypeCheck.Builtin.OneOf,
-                 TypeCheck.Builtin.PosInteger,
-                 TypeCheck.Builtin.Range,
-                 TypeCheck.Builtin.Tuple,
-                 TypeCheck.Builtin.Function,
-                ] do
+  for module <- [
+        # TypeCheck.Builtin.Any,
+        TypeCheck.Builtin.Atom,
+        TypeCheck.Builtin.Binary,
+        TypeCheck.Builtin.Bitstring,
+        TypeCheck.Builtin.Boolean,
+        TypeCheck.Builtin.CompoundFixedMap,
+        TypeCheck.Builtin.FixedList,
+        TypeCheck.Builtin.FixedMap,
+        TypeCheck.Builtin.FixedTuple,
+        TypeCheck.Builtin.Float,
+        TypeCheck.Builtin.Function,
+        # TypeCheck.Builtin.Guarded,
+        TypeCheck.Builtin.ImplementsProtocol,
+        TypeCheck.Builtin.Integer,
+        TypeCheck.Builtin.Lazy,
+        TypeCheck.Builtin.List,
+        TypeCheck.Builtin.Literal,
+        TypeCheck.Builtin.Map,
+        TypeCheck.Builtin.MaybeImproperList,
+        TypeCheck.Builtin.NamedType,
+        TypeCheck.Builtin.NegInteger,
+        TypeCheck.Builtin.None,
+        TypeCheck.Builtin.NonNegInteger,
+        TypeCheck.Builtin.Number,
+        TypeCheck.Builtin.OneOf,
+        # TypeCheck.Builtin.Pid,
+        TypeCheck.Builtin.Port,
+        TypeCheck.Builtin.PosInteger,
+        TypeCheck.Builtin.Range,
+        TypeCheck.Builtin.Reference,
+        # TypeCheck.Builtin.SizedBitstring,
+        TypeCheck.Builtin.Tuple
+      ] do
```

Comment:
Thank you for making this more clear! :green_heart:

Reply:
It's not just clarity, I've added all new modules here that were missed and commented out those of them that were failing. The rest is just `mix format`.